kernel/kernel-auth-service/src/main/java/io/mosip/kernel/auth/defaultimpl/repository/DataStore.java

```diff
@@ -64,5 +65,7 @@
 	public ValidationResponseDto validateUserName(String userId);
 
 	public UserDetailsResponseDto getUserDetailBasedOnUid(List<String> userIds);
+	
+	public VidDto getVidFromUserId(String userId,String realmID);
```

Comment:
@urviljoshi Can we rename the realmID as appID

Reply:
@LoganathanSekar7627  in service layer we get realmid from app id  it is named as appid in service layer when we send it to repo we have realm id .